lib/checkPullRequestReview.js

```diff
@@ -381,9 +382,63 @@ const handleResponseToReview = async (context) => {
   }
 };
 
+/**
+ * @param {import('probot').Context} context
+ * @param {string[]} assignees
+ * @param {string} comment
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ */
+
+//This is a function to check if there are more than 24 Hours with the Pr
+const isMaxReviewPeriodExceeded = async (review) => {
+  let timeOfReviewSubmission = review.submitted_at;
+  timeOfReviewSubmission = new Date(timeOfReviewSubmission);
+  let currentTime = new Date.now();
+  let diff = Math.abs(timeOfReviewSubmission - currentTime);
+  return diff > MAX_REVIEW_PERIOD;
+};
+
+const checkForReviewersWithPendingReview = async(context) => {
+  const allOpenPullRequests = await getAllOpenPullRequests(context);
+  await allOpenPullRequests.map(
+    async (pullRequest) => {
+      const { data: allReviews } = await context.github.pulls.listReviews(
+        context.repo({
+          pull_number: pullRequest.number,
+        })
+      );
+
+      const allReviewers = [];
```

Comment:
Maybe rename to `reviewersToPing`. `allReviewers` is somewhat conflicting with `allReviews`.

Reply:
Sure sir.